code/modules/tgui/states/heavy_vehicle.dm

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+/**
+ * tgui state: heavy_vehicle
+ *
+ * Checks that the user is inside a mech, that the mech is powered and is the right one.
+ *
+ */
+
+var/global/datum/ui_state/heavy_vehicle/heavy_vehicle_state = new
```

Comment:
GLOBAL_DATUM_INIT ?

Reply:
fucks sake i need to make all these into globs now